apps/price_pusher/src/evm/command.ts

```diff
@@ -89,50 +91,62 @@ export default {
       overrideGasPriceMultiplierCap,
       gasLimit,
       updateFeeMultiplier,
+      logLevel,
     } = argv;
 
+    const logger = pino({ level: logLevel });
+
     const priceConfigs = readPriceConfigFile(priceConfigFile);
     const priceServiceConnection = new PriceServiceConnection(
       priceServiceEndpoint,
       {
-        logger: {
-          // Log only warnings and errors from the price service client
-          info: () => undefined,
-          warn: console.warn,
-          error: console.error,
-          debug: () => undefined,
-          trace: () => undefined,
-        },
+        // Swtich to warn level if log level is info to reduce the noise
```

Comment:
i find this confusing -- this is saying that the internal price service logger only logs at "warning" if we have info logs coming out of here?

Reply:
also this code block seems to be repeated in a few places. Maybe worth extracting this out as a factory method for PriceServiceConnection (?)

Reply:
I'll update the comment. I think it's not that much given the whole command.ts is almost the same for all. I might refactor them entirely later.